doozer/doozerlib/distgit.py

```diff
@@ -1306,50 +1312,74 @@ def update_konflux_db(self, build_info, outcome, build_pipeline_url='', scratch=
             self.logger.warning('Konflux DB connection is not initialized, not writing build record to the Konflux DB.')
             return
 
-        self.logger.info('Storing build info in Konflux for Brew build %s', build_info['nvr'])
         try:
             dfp = DockerfileParser(str(self.dg_path.joinpath('Dockerfile')))
             source_repo = self.metadata.config.content.source.git.url
             commitish = dfp.labels['io.openshift.build.commit.url'].split('commit/')[-1]
 
-            rebase_repo_url, rebase_commitish = build_info['source'].split('#')
-
-            image_pullspec = build_info['extra']['image']['index']['pull'][0]
+            _, rebase_repo_url, _ = gather_git(['-C', self.distgit_dir, 'remote', 'get-url', 'origin'])
+            _, rebase_commitish, _ = gather_git(['-C', self.distgit_dir, 'rev-parse', 'HEAD'])
 
             try:
                 release_target = self.org_release.split('.')[-1]
             except KeyError:
                 release_target = ''
             el_target = release_target if release_target else f'el{self.metadata.branch_el_target()}'
```

Comment:
This isn't part of the pr but would be nice to use `isolate_el_version_in_release` here

Reply:
Done in fcd7faf8481e280412233b8051c9d22ff251b722